src/components/GuideItem.svelte

```diff
@@ -40,6 +41,7 @@
         href="{guide.url}"
         title="{guide.url}"
         target="_blank"
+        rel="noreferrer"
```

Comment:
Could you explain what the "noreferrer" is here for?

Reply:
Security measures to prevent cross-origin misusages by Svelte's standards. https://web.dev/external-anchors-use-rel-noopener/

Reply:
But all the guides are on the same domain (github.io) with the site.

Reply:
> But all the guides are on the same domain (github.io) with the site.

Indeed, locally it was between CORS issues and working fine. We're still protected over website injections nonetheless.